### PR TITLE
Table fixes

### PR DESCRIPTION
**Pull request by 5P4C3GR33D**

### CODE REVIEW

File: src/app/domains/Cart/routes/CartAll/CartAll.layout.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
+import { Container } from '@qonsoll/react-design'
 import { CartTable } from 'domains/Cart/components/table'
+import { CartCombined } from 'domains/Cart/components/combined'
 
 const CartAll = (props) => {
-  return <CartTable />
+  return (
+    <Container>
```
Comment (olenashevchuk) [inline src/app/domains/Cart/routes/CartAll/CartAll.layout.js:7]: I think, you don`t need this Container.  withLayout Component is the parent for your CartAll, and it has an own Container

File: src/app/domains/Product/components/views/ProductAdvancedView/ProductAdvancedView.template.js [hunk 2]
```diff
@@ -28,33 +28,43 @@ const productTypeMap = {
...
     editRoute: ROUTES_PATHS.REGULAR_PRODUCT_EDIT,
     layout: (props) => (
       <Row mb={4}>
-        <Col>
-          <ProgressBar value={props.categoryBalance || 0} />
+        <Col cw="auto">
+          <Typography>Reminder date</Typography>
+        </Col>
+        <Col cw="auto">
+          <Typography>{moment(props.reminderDate).format('MMM Do')}</Typography>
```
Comment (olenashevchuk) [inline src/app/domains/Product/components/views/ProductAdvancedView/ProductAdvancedView.template.js:56]: to do calculations better no in Component

File: src/app/components/Lib/Table/Table.template.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                            checked={selected.includes(row.id)}
+                            onChange={(event) =>
+                              toggleSelect(row.id, event.target.checked)
+                            }
+                          />
+                        </TableCell>
+                      )}
+                      <TableCell
+                        align="center"
+                        onClick={() => history.push(ROUTES_PATHS.MEMBER_SHOW)}>
```
Comment (GingerBoy14) [inline src/app/components/Lib/Table/Table.template.js:107]: It'll broke when i finish member, you should go to member show using his id, link `${ROUTES_PATHS.MEMBERS_ALL}/{memberId}`

File: src/app/components/Lib/Table/Table.template.js [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  },
+  wishes: {
+    multiselect: true,
+    middleCell: TABLE_CELLS[0],
+    productPath: ROUTES_PATHS.WISHES_SHOW
+  },
+  regular: {
+    multiselect: false,
+    middleCell: TABLE_CELLS[1],
+    productPath: ROUTES_PATHS.REGULAR_PRODUCT_SHOW
```
Comment (GingerBoy14) [inline src/app/components/Lib/Table/Table.template.js:34]: ...Path also later should "build" the same principle like member show

File: src/app/domains/Cart/components/combined/CartCombined.template.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+import PropTypes from 'prop-types'
+import { useForm } from 'react-hook-form'
+import { Modal, FabButton } from 'app/components/Lib'
+import { ProductSimpleForm } from 'app/domains/Product/components/forms'
+
+const CartCombined = (props) => {
+  const { title, typeModalEdit } = props
+
+  const [open, setOpen] = useState(false)
+  const form = useForm({})
```
Comment (GingerBoy14) [inline src/app/domains/Cart/components/combined/CartCombined.template.js:11]: U don't need to pass `{}` to useForm hook

File: src/app/domains/Cart/routes/CartAll/CartAll.layout.js (hunk 1 above)
Comment (GingerBoy14) [reply src/app/domains/Cart/routes/CartAll/CartAll.layout.js:7]: Yep, use <></> instead

File: src/app/domains/Product/components/views/ProductAdvancedView/ProductAdvancedView.template.js [hunk 6]
```diff
@@ -69,14 +79,16 @@ const ProductAdvancedView = (props) => {
     currency,
     assignedUser
   } = props
-  let history = useHistory()
+  const history = useHistory()
```
Comment (GingerBoy14) [inline src/app/domains/Product/components/views/ProductAdvancedView/ProductAdvancedView.template.js:82]: In react-router-dom they say that `history` is mutable soo... I don't is that correct to init `const` not `let`

File: src/app/domains/Purchase/components/list/PurchaseList/PurchaseList.template.js [hunk 7]
```diff
@@ -1,45 +1,47 @@
...
 ]
 //TODO delete mock data
 const PurchaseList = () => {
+  const history = useHistory()
   return (
     <List>
       {PURCHASED.map((item) => {
         return (
-          <Box mb={3}>
+          <Box mb={3} onClick={() => history.push(ROUTES_PATHS.PURCHASE_SHOW)}>
```
Comment (GingerBoy14) [inline src/app/domains/Purchase/components/list/PurchaseList/PurchaseList.template.js:44]: Same route rule as for user

File: src/app/domains/Purchase/components/views/PurchaseAdvancedView/PurchaseAdvancedView.template.js [hunk 8]
```diff
@@ -6,11 +6,11 @@ import { PropTypes } from 'prop-types'
...
   const { item } = props
-  const [elipsis, setElipsis] = useState(true)
+  // const [elipsis, setElipsis] = useState(true)
 
-  const changeElipsis = () => {
-    setElipsis(!elipsis)
-  }
+  // const changeElipsis = () => {
+  //   setElipsis(!elipsis)
+  // }
```
Comment (GingerBoy14) [inline src/app/domains/Purchase/components/views/PurchaseAdvancedView/PurchaseAdvancedView.template.js:13]: Remove comments, if it's not necessary

File: src/app/domains/Wish/components/combined/WishCombined.template.js [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+import PropTypes from 'prop-types'
+import { useForm } from 'react-hook-form'
+import { Modal, FabButton } from 'app/components/Lib'
+import { ProductSimpleForm } from 'app/domains/Product/components/forms'
+
+const WishCombined = (props) => {
+  const { title, typeModalEdit } = props
+
+  const [open, setOpen] = useState(false)
+  const form = useForm({})
```
Comment (GingerBoy14) [inline src/app/domains/Wish/components/combined/WishCombined.template.js:11]: Read previous comment for the same situation

File: src/app/domains/Wish/routes/WishAll/WishAll.layout.js [hunk 10]
```diff
@@ -1,7 +1,14 @@
+import { Container } from '@qonsoll/react-design'
 import { WishTable } from 'app/domains/Wish/components/table/'
+import { WishCombined } from 'domains/Wish/components/combined'
 
 const WishAll = (props) => {
-  return <WishTable />
+  return (
+    <Container>
```
Comment (GingerBoy14) [inline src/app/domains/Wish/routes/WishAll/WishAll.layout.js:7]: Same as in CartAll

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

**Review by AlexFry123 — APPROVED** [review]

**Review by okrdima — APPROVED** [review]

**Review by GingerBoy14 — APPROVED** [review]

**Pull request merged**